Sources/Commands/SwiftTestTool.swift

```diff
@@ -652,61 +641,48 @@ final class TestRunner {
         return args
     }
 
-    /// Executes the tests without printing anything on standard streams.
-    ///
-    /// - Returns: A tuple with first bool member indicating if test execution returned code 0
-    ///            and second argument containing the output of the execution.
-    private func test(testAt testPath: AbsolutePath) -> (Bool, String) {
-        var output = ""
-        var success = false
-        do {
-            // FIXME: The environment will be constructed for every test when using the
-            // parallel test runner. We should do some kind of caching.
```

Comment:
took care of this FIXME while at it, now the env is constructed once and injected by the caller